apps/chat-e2e/src/tests/chatApi/assistantPlusAddons.test.ts

```diff
@@ -0,0 +1,31 @@
+import { skipReason } from '@/src/core/baseFixtures';
+import dialTest from '@/src/core/dialFixtures';
+import { AssistantPlusAddonsRequest } from '@/src/testData';
+
+const entityPlusAddonsRequests = process.env.ASSISTANT_PLUS_ADDONS_FOR_API_TESTS
+  ? (JSON.parse(
+      process.env.ASSISTANT_PLUS_ADDONS_FOR_API_TESTS,
```

Comment:
it relies on the ASSISTANT_PLUS_ADDONS_FOR_API_TESTS environment variable being correctly configured. what if variable is missing or invalid?
Same questions for other similar situations

Reply:
updated readme with the comment